lightning/src/onion_message/messenger.rs

```diff
@@ -488,7 +491,7 @@ where
 				message, control_tlvs: ReceiveControlTlvs::Unblinded(ReceiveTlvs { path_id }), reply_path,
 			}, None)) => {
 				log_trace!(self.logger,
-					"Received an onion message with path_id {:02x?} and {} reply_path",
+						"Received an onion message {:?} with path_id {:02x?} and {} reply_path", message,
```

Comment:
Let's similarly put the message at the end (it's gonna be large) instead of in the middle of the message. Also, let's fix this and the line below to indent only once.